Formula/projectm.rb

```diff
@@ -0,0 +1,22 @@
+class Projectm < Formula
+  desc "Milkdrop-compatible music visualizer. Standalone app and Music.app plugin"
```

Comment:
It looks like homebrew-casks fits better for this app.

https://docs.brew.sh/Acceptable-Formulae#stuff-that-builds-an-app

Reply:
I think the description is inaccurate for the source build here, I'll update it. It builds a command-line app.

<img width="1545" alt="Screen Shot 2021-02-14 at 6 17 05 PM" src="https://user-images.githubusercontent.com/245131/107884833-bbb74c80-6eff-11eb-9a95-aa60cca0a768.png">